datarepo-clienttests/src/main/resources/configs/LookupSnapshot.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "LookupSnapshot",
+  "description": "Run lookups on a snapshot.",
```

Comment:
Nit: This is running lookups on two snapshots because there are two elements in the testScripts array below.